crates/solidity/outputs/cargo/build/src/main.rs

```diff
@@ -12,13 +12,8 @@ use solidity_language::SolidityLanguageExtensions;
 // 3) We want to avoid having dependencies from the source crate to codegen crates.
 //
 fn main() -> Result<()> {
-    let language = LanguageDefinition::load_solidity()?;
-
+    let grammar = Grammar::new();
```

Comment:
I think this is generating the NAPI sources twice, under both the rust and the NPM crate. Is that intended? Not sure what happened to the `generate_rust_*` and `generate_napi_*` separation?

Reply:
Source is identical, but gated via the feature flag. No need to separate them.

Reply:
Right now on `main`, it is also using a single source for both, but the npm crate is importing the earlier one via `#[path]`. I wonder what changed? why do we need the second copy if it is identical?

That means twice the work to do during the build, and slightly more effort during code review.

Reply:
This solution is simple, and works with napi building. The correct solution is a common crate being used with different features in the target crates. But my experience with napi building was impossible to get right.

Reply:
I see. Sounds good to me. In that case, how about adding a `crates/solidity/outputs/npm/crate/src/.gitignore` file with the following contents:

```ignore
# Contents of this generated folder are an exact duplicate of 'crates/solidity/outputs/cargo/crate/src/generated'.
# Let's ignore it here, so that we don't end up reviewing them twice every time the output changes, both during development and in code reviews.
./generated/**
```

IIUC, this should not have any effect on NAPI, Cargo build scripts, or Rust Analyzer intellisense.
Thoughts? I'm also happy to do it later if you like the idea.

Reply:
Sounds good. Thanks!